bindings/python/src/PyXRootDFile.cc

```diff
@@ -106,14 +106,14 @@ namespace PyXRootD
   PyObject* File::Stat( File *self, PyObject *args, PyObject *kwds )
   {
     static const char  *kwlist[] = { "force", "timeout", "callback", NULL };
-    bool                force    = false;
+    int                 force    = 0;
     uint16_t            timeout  = 0;
     PyObject           *callback = NULL, *pyresponse = NULL, *pystatus = NULL;
     XrdCl::XRootDStatus status;
 
     if ( !self->file->IsOpen() ) return FileClosedError();
 
-    if ( !PyArg_ParseTupleAndKeywords( args, kwds, "|iHO:stat", (char**) kwlist,
+    if ( !PyArg_ParseTupleAndKeywords( args, kwds, "|pHO:stat", (char**) kwlist,
```

Comment:
I think p here is python 3.3+, if you still need python 2 then it probably needs to be kept as i and checked for truthiness on the python side.

Reply:
Ok, thanks, I will check it when testing.

Reply:
I think we should only change `bool` to `int` above, while leaving `iHO` as is to not lose compatibility with earlier versions of Python. I tested it and it works. Once this change is made we can merge it. Thanks!